diff -u libharu-2.0.8.orig/configure libharu-2.0.8/configure
--- libharu-2.0.8.orig/configure	2008-12-26 17:50:32.168746495 +0100
+++ libharu-2.0.8/configure	2008-12-26 17:58:36.905781205 +0100
@@ -271,21 +271,21 @@
 	# commands to install #
 	echo "INCDIR=${PREFIX}/include"
 	echo "LIBDIR=${PREFIX}/lib"
-	INSTALL_CMD0='if [ ! -d $(PREFIX) ]; then mkdir -p $(PREFIX); fi'
-	INSTALL_CMD1='if [ ! -d $(PREFIX)/include ]; then mkdir -p $(PREFIX)/include; fi'
-	INSTALL_CMD2='if [ ! -d $(PREFIX)/lib ]; then mkdir -p $(PREFIX)/lib; fi'
+	INSTALL_CMD0='if [ ! -d $(DESTDIR)$(PREFIX) ]; then mkdir -p $(DESTDIR)$(PREFIX); fi'
+	INSTALL_CMD1='if [ ! -d $(DESTDIR)$(PREFIX)/include ]; then mkdir -p $(DESTDIR)$(PREFIX)/include; fi'
+	INSTALL_CMD2='if [ ! -d $(DESTDIR)$(LIBDIR) ]; then mkdir -p $(DESTDIR)$(LIBDIR); fi'
 	if test $SHARED = yes; then
-		INSTALL_CMD3='cp include/hpdf.h include/hpdf_consts.h include/hpdf_types.h $(PREFIX)/include/; chmod 644 $(PREFIX)/include/hpdf*.h'
-		INSTALL_CMD4='cp -p $(SONAME)$(SOVER1)$(SOVER2) $(PREFIX)/lib/'
+		INSTALL_CMD3='cp include/hpdf.h include/hpdf_consts.h include/hpdf_types.h $(DESTDIR)$(PREFIX)/include/; chmod 644 $(DESTDIR)$(PREFIX)/include/hpdf*.h'
+		INSTALL_CMD4='cp -p $(SONAME)$(SOVER1)$(SOVER2) $(DESTDIR)$(LIBDIR)'
 		if test $SHARED_EXT != '.dll'; then
-			INSTALL_CMD5='ln -sf $(SONAME)$(SOVER1)$(SOVER2) $(PREFIX)/lib/$(SONAME)$(SOVER1)'
-			INSTALL_CMD6='ln -sf $(SONAME)$(SOVER1) $(PREFIX)/lib/$(SONAME)'
+			INSTALL_CMD5='ln -sf $(SONAME)$(SOVER1)$(SOVER2) $(DESTDIR)$(LIBDIR)/$(SONAME)$(SOVER1)'
+			INSTALL_CMD6='ln -sf $(SONAME)$(SOVER1) $(DESTDIR)$(LIBDIR)/$(SONAME)'
 		else
-			INSTALL_CMD5='cp -p $(LIBNAME) $(PREFIX)/lib/'
+			INSTALL_CMD5='cp -p $(LIBNAME) $(DESTDIR)$(LIBDIR)/'
 		fi
 	else
-		INSTALL_CMD3='cp include/hpdf*.h $(PREFIX)/include/; chmod 644 $(PREFIX)/include/hpdf*.h'
-		INSTALL_CMD4='cp -p $(LIBNAME) $(PREFIX)/lib/'
+		INSTALL_CMD3='cp include/hpdf*.h $(DESTDIR)$(PREFIX)/include/; chmod 644 $(DESTDIR)$(PREFIX)/include/hpdf*.h'
+		INSTALL_CMD4='cp -p $(LIBNAME) $(DESTDIR)$(LIBDIR)/'
 	fi	
 fi
 
Wspólne podkatalogi: libharu-2.0.8.orig/demo i libharu-2.0.8/demo
Wspólne podkatalogi: libharu-2.0.8.orig/doc i libharu-2.0.8/doc
Wspólne podkatalogi: libharu-2.0.8.orig/if i libharu-2.0.8/if
Wspólne podkatalogi: libharu-2.0.8.orig/include i libharu-2.0.8/include
Wspólne podkatalogi: libharu-2.0.8.orig/script i libharu-2.0.8/script
Wspólne podkatalogi: libharu-2.0.8.orig/src i libharu-2.0.8/src
